src/OrbitGl/CGroupAndProcessMemoryTrack.cpp

```diff
@@ -52,6 +52,17 @@ CGroupAndProcessMemoryTrack::CGroupAndProcessMemoryTrack(
   const std::string kValueLowerBoundLabel = "Minimum: 0 GB";
   constexpr double kValueLowerBoundRawValue = 0.0;
   TrySetValueLowerBound(kValueLowerBoundLabel, kValueLowerBoundRawValue);
+
+  const std::string kGameCGroupName = "game";
```

Comment:
Add some comment explaining this constant

Reply:
Done! Thanks!